userguide/system.rst

```diff
@@ -2288,21 +2288,51 @@ Updating an HA System
 
 If the %brand% array has been configured for High Availability
 (HA), the update process must be started on the active |ctrlr-term|.
-Once the update is complete, the standby |ctrlr-term| will automatically
-reboot. Wait for it to come back up by monitoring the remote console or
-the |web-ui| of the standby |ctrlr-term|.
+The update process is to download and synchronize the update files
+between both |ctrlrs-term|, update and reboot the standby |ctrlr-term|,
```

Comment:
Not clear what "synchronize between" means.  I think this just means download the update files to both controllers.  That the update might do it for the user is, as they say, an implementation detail.